ValveAnnulusAnalysis/HeartValveWidgets/ValveSequenceBrowserWidget.py

```diff
@@ -0,0 +1,234 @@
+import logging
+from string import Template
+from enum import IntEnum
+
+import vtk
+import slicer
+from slicer.util import VTKObservationMixin
+
+import HeartValveLib
+from HeartValveLib.Constants import CARDIAC_CYCLE_PHASE_PRESETS
+
+
+MODULE_NAME = "ValveAnnulusAnalysis"
+
+
+def resourcePath(filename):
+  import os
+  import slicer
+  scriptedModulesPath = os.path.dirname(slicer.util.modulePath(MODULE_NAME))
+  return os.path.join(scriptedModulesPath, 'Resources', filename)
+
+
+class ValveSequenceBrowserWidget:
```

Comment:
It would be nice to add a very little documentation to the class, just 1-2 sentences to describe what the widget is for and what properties are required to be set.